packages/docs/content/components/text-field/text-field.mdx

```diff
@@ -37,6 +37,10 @@ A `TextField` component renders an input field as well as supporting UI elements
 
 <SeeStorybookForGuidance storyId={'components-textfield--docs'} />
 
+### Web Component
+
+<SeeStorybookForGuidance tech="wc" storyId={'web-components-ds-text-field--docs'} />
```

Comment:
Is there a plan to include the same sort of links on the `text-field/label-masked-field` and `text-field/masked-field` pages? Would it require making label-mask stories like the React component has, or would they just link to the root `ds-text-field` Storybook doc page?

Reply:
Thanks for the feedback, @pwolfert! In the React section of this MDX file, I didn’t notice any specific links to distinct stories for label-masked-field and masked-field—it just links back to the main docs page in storybook. Following that structure, I planned to do the same for the web component section.

But, I could see how having dedicated web component stories for label-masked-field and masked-field might be valuable. This way, users wouldn’t have to infer that functionality solely from the react component’s storybook documentation.